### ISSUE TITLE
Hot design missing when appBuilder (on linux)

### ISSUE THREAD

**Issue by sarosigy** [unoplatform/uno.extensions#2655]
### Current behavior

I just created a brand new Uno desktop app, it runs as it should.

I wanted to leverage the Localization which means using Hosting, also. I did exactly what is written in [Uno Howto Hosting Setup](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Hosting/HowTo-HostingSetup.html)

But even adding the single line `var appBuilder = this.CreateBuilder(args);` causes two things:
1. The Hot Design toolbar won't show on the Main Window
2. The application would not exit by closing with the Close button, it must be stopped in debugger (or its process killed)

### Expected behavior

It should work as expected.

### How to reproduce it (as minimally and precisely as possible)

1. `dotnet new unoapp --platforms desktop -o unoinout`
2. Add `Hosting` to `<UnoFeatures>`
3. Add the line `var appBuilder = this.CreateBuilder(args);` in the first line of App.xaml.cs `protected override void OnLaunched(LaunchActivatedEventArgs args)`

### Workaround

Not using appBuilder.

### Which tool(s) are affected?

Hot Design™

### Affected platforms

Desktop (Skia)

### Uno.Sdk version

5.5.54

### IDE used

Visual Studio Code on Linux

### IDE version

1.96.0

### Uno Platform Extension version

0.18.5

**Comment by eriklimakc** [comment]
@sarosigy @nickrandolph The details about changing how the `MainWindow` is assigned can be found here:  
[Installing Extensions in an Existing Project](https://platform.uno/docs/articles/external/uno.extensions/doc/GettingStarted.html?tabs=wizard#installing-extensions-in-an-existing-project).  

I've submitted a [PR](https://github.com/unoplatform/uno.extensions/pull/2673) with some improvements to the docs to enhance clarity and add more details.

**Comment by sarosigy** [comment]
> [@sarosigy](https://github.com/sarosigy) [@nickrandolph](https://github.com/nickrandolph) The details about changing how the `MainWindow` is assigned can be found here: [Installing Extensions in an Existing Project](https://platform.uno/docs/articles/external/uno.extensions/doc/GettingStarted.html?tabs=wizard#installing-extensions-in-an-existing-project).
> 
> I've submitted a [PR](https://github.com/unoplatform/uno.extensions/pull/2673) with some improvements to the docs to enhance clarity and add more details.

Thank you @eriklimakc, it will add more clarity.

I maybe would add that ```MainWindow = appBuilder.Window;``` line to the HostingOverview.md, too, because that Hosting feature requires to change the default generated line, and not everyone (including myself) would read the general overview when adding a single feature. Maybe it is our limitation but it would be nice to handle this behavior.

**Comment by eriklimakc** [comment]
> Thank you [@eriklimakc](https://github.com/eriklimakc), it will add more clarity.
> 
> I maybe would add that `MainWindow = appBuilder.Window;` line to the HostingOverview.md, too, because that Hosting feature requires to change the default generated line, and not everyone (including myself) would read the general overview when adding a single feature. Maybe it is our limitation but it would be nice to handle this behavior.

@sarosigy Thanks for the thoughtful feedback! I’ve updated the PR to include your suggestion.

**Comment by Darsh0307** [comment]
@eriklimakc  With 5.6.17 as the Uno.sdk i was easily able to test it on a blank app. 
And there wasnt any such problem occuring.

**Comment by sarosigy** [comment]
@Darsh0307 I believe that you have the ```MainWindow = appBuilder.Window;``` line in your code.
If you create your App without hosting, that line will not be there, instead it will be ```MainWindow = new Window();```

We agreed that it is a documentation issue because if it would be in the documentation in the first place the whole issue would be nonexistent.

This is why this case is closed already.

**Comment by Darsh0307** [comment]
@sarosigy  

You are correct , I have `MainWindow = appBuilder.Window` in my code as you said.
